.storybook/styles.css

```diff
@@ -7,29 +7,29 @@
   line-height: var(--salt-text-lineHeight);
 }
 
-.salt-theme .sbdocs h1 {
+.salt-theme .sbdocs h1:not(.saltText) {
   color: var(--salt-text-primary-foreground);
   font-size: var(--salt-text-h1-fontSize);
   font-weight: var(--salt-text-h1-fontWeight);
   line-height: var(--salt-text-h1-lineHeight);
 }
 
-.salt-theme .sbdocs h2 {
+.salt-theme .sbdocs h2:not(.saltText) {
   border-bottom: none;
```

Comment:
was there a border-bottom on h2? just curious

Reply:
in Storybook default styling yes